### PR TITLE
Add new insurance funding rate + add tests

### PR DESCRIPTION
**Pull request by kumar-ish**
# Motivation

Since the original whitepaper for Tracer's perpetual swaps was released, the way that the insurance funding rate is calculated has been changed. Below is the new formula:
image

Sorry for my chopped-up three second LaTeX :((

# Changes

- Update the way that the insurance funding rate is calculated and update corresponding unit tests

### CODE REVIEW

File: test/unit/Insurance.js [hunk 1]
```diff
@@ -629,26 +630,68 @@ describe("Unit tests: Insurance.sol", function () {
             })
         })
 
-        context("when the leveraged notional value is > 0", async () => {
-            it("returns the appropriate 8 hour funding rate", async () => {
+        context("when the pool is greater than the target", async () => {
```
Comment (CalabashSquash) [inline test/unit/Insurance.js:633]: Would you be able to write a test for when the pool is < the target && `(((fund_target - fund_holdings)/fund_target) ** 2) < 1`?

There are tests for the two edge cases (pool above target & pool has no holdings), but just for peace of mind a normal case test would be good.

**Review by CalabashSquash — CHANGES_REQUESTED** [review]

File: contracts/Insurance.sol [hunk 2]
```diff
@@ -221,7 +216,7 @@ contract Insurance is IInsurance {
             return 0;
         }
 
-        uint256 ratio = PRBMathUD60x18.div(poolTarget - poolHoldings, levNotionalValue);
+        uint256 ratio = PRBMathUD60x18.pow(PRBMathUD60x18.div(poolTarget - poolHoldings, poolTarget), 2 * 10**18);
```
Comment (CalabashSquash) [inline contracts/Insurance.sol]: Also, RBMath.pow(x, y) requires x > 1 in WAD (see https://github.com/hifi-finance/prb-math/blob/main/contracts/PRBMathUD60x18.sol#L357).

Since poolTarget - poolHoldings / poolTarget is always < 1, I'm not sure how this works. A test case as suggested in my other comment would check this.

**Review by CalabashSquash — CHANGES_REQUESTED** [review]

File: contracts/Insurance.sol [hunk 3]
```diff
@@ -200,18 +200,13 @@ contract Insurance is IInsurance {
     /**
      * @notice Gets the 8 hour funding rate for an insurance pool
      * @dev the funding rate is represented as
-     *      0.0036523 * (insurance_fund_target - insurance_fund_holdings) / leveraged_notional_value)
+     *      (182.648 / 8) * (5 * ((fundTarget - fundHoldings) / (fundTarget * 10_000))) ** 2
```
Comment (greenbookwebb) [inline contracts/Insurance.sol:203]: This could throw an error. div by 0/ Fund holdings could be positive

**Review by greenbookwebb — COMMENTED** [review]
Looks good overall. I didn't go into the tests though. Let's just talk in person about tests if you want me to discuss it.

File: contracts/Insurance.sol (hunk 3 above)
Comment (CalabashSquash) [reply contracts/Insurance.sol:203]: It could by this alone, but below there is a check for
```
        if (poolTarget <= poolHoldings) {
            return 0;
        }
```
if `poolTarget` is 0, then poolHoldings must be at least == pool target, and therefore will enter this branch and return 0

File: test/unit/Insurance.js (hunk 1 above)
Comment (kumar-ish) [reply test/unit/Insurance.js:633]: Done

**Review by CalabashSquash — APPROVED** [review]

**Review by raymogg — APPROVED** [review]
